tests/sentry/eventstream/kafka/test_consumer.py

```diff
@@ -24,37 +24,42 @@
 
 from django.conf import settings
 
-settings.KAFKA_CLUSTERS["default"] = {
-    "common": {"bootstrap.servers": os.environ.get("SENTRY_KAFKA_HOSTS", "localhost:9092")}
-}
+SENTRY_KAFKA_HOSTS = os.environ.get("SENTRY_KAFKA_HOSTS", "127.0.0.1:9092")
+SENTRY_ZOOKEEPER_HOSTS = os.environ.get("SENTRY_ZOOKEEPER_HOSTS", "sentry_zookeeper:2181")
```

Comment:
`SENTRY_ZOOKEEPER_HOSTS=127.0.0.1:2181 SENTRY_KAFKA_HOSTS=127.0.0.1:9092 pytest tests/sentry/eventstream/kafka/test_consumer.py` was just completely wrong before. This works locally but not in CI though...